src/argus_htmx/static/styles.css

```diff
@@ -0,0 +1,3677 @@
+/*
+! tailwindcss v3.4.10 | MIT License | https://tailwindcss.com
+*/
+
+/*
+1. Prevent padding and border from affecting element width. (https://github.com/mozdevs/cssremedy/issues/4)
+2. Allow adding a border to an element by just adding a border-width. (https://github.com/tailwindcss/tailwindcss/pull/116)
+*/
```

Comment:
You probably already know and this is most likely due to pushing as early as you did, but you probably don't want to check in a compiled tailwind css -> .gitignore